lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -188,8 +188,14 @@ def self.fetch_all_tags
       #
       def self.point_to_same_commit?(ref1, ref2)
         git_repo = Git.open(Dir.pwd)
-        ref1_commit = git_repo.gcommit(ref1)
-        ref2_commit = git_repo.gcommit(ref2)
+
+        begin
+          ref1_commit = git_repo.gcommit(ref1)
+          ref2_commit = git_repo.gcommit(ref2)
+        rescue StandardError => e
+          puts "Error: #{e.message}"
```

Comment:
What do you think of:

```suggestion
          UI.error "Error fetching commits for #{ref1} and #{ref2}: #{e.message}"
```

Actually, those are two different suggestions:

- Use `UI.error` instead of `puts`
- Include the attempted operation in the error message